$NetBSD: patch-aa,v 1.1.1.1 2006/07/23 11:50:25 rillig Exp $

Since some systems (among them NetBSD 3.0) don't have a log2f function,
precompute the values.

--- plugins/organic/organic.cpp.orig	2006-07-03 08:34:22.000000000 +0200
+++ plugins/organic/organic.cpp	2006-07-23 12:07:40.000000000 +0200
@@ -106,14 +106,14 @@ organicInstrument::organicInstrument( in
 
 	m_osc = new oscillatorData[m_num_oscillators];
 	
-	m_osc[0].harmonic = log2f( 0.5f );	//	one octave below
-	m_osc[1].harmonic = log2f( 0.75f );	//	a fifth below
-	m_osc[2].harmonic = log2f( 1.0f );	// base freq
-	m_osc[3].harmonic = log2f( 2.0f );	// first overtone
-	m_osc[4].harmonic = log2f( 3.0f );	// second overtone
-	m_osc[5].harmonic = log2f( 4.0f );	// .
-	m_osc[6].harmonic = log2f( 5.0f );	// .
-	m_osc[7].harmonic = log2f( 6.0f );	// .
+	m_osc[0].harmonic = -1.0000000f;	//	one octave below
+	m_osc[1].harmonic = -0.4150375f;	//	a fifth below
+	m_osc[2].harmonic =  0.0000000f;	// base freq
+	m_osc[3].harmonic =  1.0000000f;	// first overtone
+	m_osc[4].harmonic =  1.5849625f;	// second overtone
+	m_osc[5].harmonic =  2.0000000f;	// .
+	m_osc[6].harmonic =  2.3219281f;	// .
+	m_osc[7].harmonic =  2.5849625f;	// .
 
 	for (int i=0; i < m_num_oscillators; i++)
 	{
